src/OpenTelemetry.Extensions/OpenTelemetry.Extensions.csproj

```diff
@@ -21,6 +21,12 @@
 
   <ItemGroup>
     <PackageReference Include="OpenTelemetry" Version="$(OpenTelemetryCoreLatestVersion)" />
+    <PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="$(OpenTelemetryCoreLatestVersion)" />
+    <PackageReference Include="System.Text.Json" Version="9.0.4" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Remove="nupkgs\OpenTelemetry.Extensions.1.11.0-beta.1.67.snupkg" />
```

Comment:
Can you explain a little more about why we need this?

Reply:
probably a leftover from testing. it will be removed.